python/dglke/models/general_models.py

```diff
@@ -493,13 +501,8 @@ def forward(self, pos_g, neg_g, gpu_id=-1):
         """
         pos_g.ndata['emb'] = self.entity_emb(pos_g.ndata['id'], gpu_id, True)
         pos_g.edata['emb'] = self.relation_emb(pos_g.edata['id'], gpu_id, True)
-
         self.score_func.prepare(pos_g, gpu_id, True)
-
         pos_score = self.predict_score(pos_g)
-        pos_score = logsigmoid(pos_score)
-        if self.has_edge_importance:
```

Comment:
Where do you handle edge importance?
We should support this feature.
The importance weight is applied to both pos and neg score in a pairwise manner.